src/routes/[lang]/contact/christian/+page.svelte

```diff
@@ -0,0 +1,62 @@
+<script lang="ts">
+  import { browser } from '$app/environment'
+  import googleWallet from '$lib/assets/google-wallet.svg'
+  import iosWallet from '$lib/assets/ios-wallet.svg'
+
+  export let data
+  const { passGoogle, pkpass } = data
+
+  let isIOS: boolean
+  let isAndroid: boolean
+  let isMac: boolean
+
+  if (browser) {
+    const { userAgent } = navigator
+    isIOS = /(iPhone|iPad|iPod)/.test(userAgent)
+    isAndroid = /(Android)/.test(userAgent)
+    isMac = /(Macintosh)/.test(userAgent)
+
+    if (isIOS || isMac) {
+      window.location.href = pkpass
+    } else if (isAndroid) {
+      window.location.href = passGoogle
+    }
+  }
+</script>
+
+<div class="add-wallet">
+  <a href={pkpass}
+    ><img class="link-wallet" src={iosWallet} alt="button-ios" /></a
```

Comment:
bad indentation
img tag is not a link either, so class name is wrong - it's best if it describes what it is so code makes sense... use "wallet-logo" for example
please study alt attribute and correct